js/version-visibility.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/BUILD.bazel b/tools/BUILD.bazel
+index 6f86f0a..3058f94 100644
+--- a/tools/BUILD.bazel
++++ b/tools/BUILD.bazel
+@@ -17,7 +17,7 @@ bzl_library(
+ bzl_library(
+     name = "version",
+     srcs = [":version.bzl"],
+-    visibility = ["//lib/private/docs:__pkg__"],
++    visibility = ["//visibility:public"],
```

Comment:
This is already fixed in 2.0 but it's not in the latest 1.x release. We could do another release there rather than include this patch if you prefer.

Reply:
So an issue is that `versions.bzl` is in a package that has go deps, which makes this more difficult.